pyfive/h5d.py

```diff
@@ -232,18 +234,19 @@ def convert_slice(aslice):
     ##### bypass the iterator methods.
     @property
     def index(self):
-        """ Direct access to the chunk index, if there is one."""
+        """ Direct access to the chunk index, if there is one. This is a ``pyfive`` API extension. """
         if self._index is None:
             raise ValueError('No chunk index available for HDF layout class {self.layout}')
         else:
             return self._index
     #### The following method can be used to set pseudo chunking size after the 
     #### file has been closed and before data transactions. This is pyfive specific
     def set_psuedo_chunk_size(self, newsize_MB):
```

Comment:
```suggestion
    def set_pseudo_chunk_size(self, newsize_MB):
```

Reply:
There might be more occurences?

Reply:
Sadly you're probably right, but we are hoping to do a wee internal hackathon in autumn, where we might deal with both enums and our test coverage ... so if they're not picked up before then, hopefully proper coverage tests will find the rest.